test/test_yaml.py

```diff
@@ -73,3 +73,28 @@ def test_empty_note_in_yaml_input(self):
         empty_note = "First name: foo\nNote:"
         x = self._parse_yaml(empty_note)
         self.assertListEqual(x.notes, [])
+
+class yaml_ablabel(unittest.TestCase):
+
+    @staticmethod
+    def _parse_yaml(yaml=''):
+        """Parse some yaml string into a CarddavObject
+
+        :param yaml: the yaml input string to parse
+        :type yaml: str
+        :returns: the parsed CarddavObject
+        :rtype: CarddavObject
+        """
+        # Careful, this function doesn't actually support named arguments so
+        # they have to be kept in this order!
+        return CarddavObject.from_user_input(
+                address_book=mock.Mock(path='foo-path'), user_input=yaml,
+                supported_private_objects=[], version='3.0',
+                localize_dates=False)
+
+    def test_ablabelled_url_in_yaml_input(self):
+        ablabel_url = "First name: foo\nWebpage:\n - http://example.com\n" +\
```

Comment:
You can use quotes in YAML to disambiguate a string from a dict, so 
```yaml
Webpage:
 - "http://example.com"
 - github: "https://github.com/scheibler/khard"
```
should work.  But I though YAML only uses colons without a following space for dicts. So is there actually a syntax problem in your YAML or not?

And you don't need the `+` here as Python has implicit concatenation of string *literals*. That means `"foo"  "bar" == "foobar"`

Reply:
> You can use quotes in YAML to disambiguate a string from a dict, so
> 
> should work. But I though YAML only uses colons without a following space for dicts. So is there actually a syntax problem in your YAML or not?

I was trying to make it reflect the behaviour of `khard edit`, which gives the URL as `https\://`, possibly to avoid the unlabelled URL becoming a dict. It certainly interprets correctly in-app.

The test works with or without quotes - I can add them if you want.

> And you don't need the `+` here as Python has implicit concatenation of string _literals_. That means `"foo" "bar" == "foobar"`

Ah. I didn't realise that worked across lines (with `\`)